README.md

```diff
@@ -165,3 +170,8 @@ Short description.
 ## License
 
 This work and all contributions to it are released into the public domain under the terms of the [CC0 1.0](./LICENSE).
+<!--stackedit_data:
```

Comment:
I'm not sure what this is,but I don't think it's meant to be here :smile:

Reply:
Had to delink this repo from Stackedit, even after I deleted this line from the Readme on Github itself, it was auto-syncing and adding it again and again. Sorry about that.